{
  "original_file": "c:\\Users\\patra\\Genesis FINAL TRY\\modules\\unclassified\\phase_89_execution_validation.py",
  "quarantine_timestamp": "2025-06-21T04:33:38.393589+00:00",
  "violations": [
    {
      "rule_type": "no_mock_data",
      "pattern": "test_",
      "file": "c:\\Users\\patra\\Genesis FINAL TRY\\modules\\unclassified\\phase_89_execution_validation.py",
      "line_numbers": [
        37,
        39,
        41,
        42,
        44
      ],
      "violation_count": 22
    },
    {
      "rule_type": "no_fallback_logic",
      "pattern": "try:",
      "file": "c:\\Users\\patra\\Genesis FINAL TRY\\modules\\unclassified\\phase_89_execution_validation.py",
      "line_numbers": [
        25,
        81,
        136
      ],
      "violation_count": 3
    },
    {
      "rule_type": "no_fallback_logic",
      "pattern": "except Exception",
      "file": "c:\\Users\\patra\\Genesis FINAL TRY\\modules\\unclassified\\phase_89_execution_validation.py",
      "line_numbers": [
        30,
        130,
        146
      ],
      "violation_count": 3
    },
    {
      "rule_type": "no_fallback_logic",
      "pattern": "if not",
      "file": "c:\\Users\\patra\\Genesis FINAL TRY\\modules\\unclassified\\phase_89_execution_validation.py",
      "line_numbers": [
        56,
        59,
        62,
        68,
        71
      ],
      "violation_count": 6
    },
    {
      "rule_type": "no_fallback_logic",
      "pattern": "else:",
      "file": "c:\\Users\\patra\\Genesis FINAL TRY\\modules\\unclassified\\phase_89_execution_validation.py",
      "line_numbers": [
        124
      ],
      "violation_count": 1
    }
  ],
  "quarantine_reason": "ARCHITECT_MODE_VIOLATION"
}